SalesforceFoundation/Cumulus
# Feature/gift entry  form save

# Critical Changes

# Changes
 * add support for Saving the form rendered

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Instructions
* create new template with the Template builder. Name the template 'Single Gift Entry Template'. Use the url part /lightning/cmp/c__geTemplateBuilderPlaceholder appended to your instance url.
* Open the Form Render with the url part /lightning/cmp/c__geFormRendererPlaceholder appended to your instance url.
* Check if the fields in the Template are loaded.
* Enter data and Save
* Check if all the populated fields saved  to the opportunity and related objects.

# Not Included
* Lookup fields are not supported at this time
